✨ :sparkles: feat(workflows): Add workflow telemetry for better monitoring

TGTGamer opened the pull request:
This commit adds the workflow telemetry feature to the GitHub workflows. The `catchpoint/workflow-telemetry-action` v1.8.7 action is used to integrate telemetry functionality. The action is configured to enable system process tracing, dark theme, and commenting on pull requests. This addition will provide better insights and monitoring for the workflows.

✨ **:sparkles: feat(workflows)**: Resolve component packages from bit.cloud registry

This commit introduces a step in the workflows to resolve component packages from the bit.cloud registry. The step configures npm to use the bit.cloud registry and sets the authentication token. This change is necessary for component installation using package managers other than Bit.

💄 **:lipstick: style(workflows)**: Remove unnecessary conditional check in CLA Assistant

This commit removes an unnecessary conditional check in the CLA workflow. The conditional check on the `cla_enabled` environment variable is removed, as it is redundant and does not affect the workflow's functionality.

♻️ **:recycle: refactor(workflows)**: Cleanup and standardize workflow files

This commit standardizes and cleans up various workflow files. The changes ensure consistency and improve readability across the workflow files. No functional changes are included in this commit.

🔨 **:hammer: fix(workflows)**: Update file modification date in GitHub Management workflow

This commit updates the last modified date in the header comment of the GitHub Management workflow file. The modification date is corrected to reflect the actual date of the file modification.

🐛 **:bug: fix(workflows)**: Use correct workflow telemetry action in GitGuardian IAC scan

This commit fixes a mistake in the GitGuardian IAC scan workflow. The workflow telemetry action was using an incorrect version. The action is now correctly set to `catchpoint/workflow-telemetry-action@v1.8.7` for better monitoring and insights.

🎨 **:art: style(workflows)**: Update header comment in Lane Cleanup workflow

This commit updates the header comment in the Lane Cleanup workflow file to include relevant information. The header comment now includes the project, file path, author information, and license details. The modification date is also updated to reflect the latest changes.


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Implemented Workflow Telemetry across various GitHub Actions workflows for enhanced monitoring and insights.
  - Updated the README.md with new sections for Contributing, Visualization of the Codebase, and Backlog information.
  - Added a feature to resolve component packages from specific registries, improving dependency management.

- **Documentation**
  - Enhanced the README.md with images and detailed project stack information.
  - Removed the Security section from the README.md for streamlining documentation.

- **Bug Fixes**
  - Fixed the conditional execution of the CLA Assistant step to now run unconditionally.

- **Chores**
  - Updated "Last Modified" dates in workflow files to reflect recent changes.

- **Refactor**
  - Standardized the addition of file header comments across workflow files for better project information and guidelines.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

TGTGamer committed:
✨ **:sparkles: feat(workflows)**: Add workflow telemetry for better monitoring

This commit adds the workflow telemetry feature to the GitHub workflows. The `catchpoint/workflow-telemetry-action` v1.8.7 action is used to integrate telemetry functionality. The action is configured to enable system process tracing, dark theme, and commenting on pull requests. This addition will provide better insights and monitoring for the workflows.

✨ **:sparkles: feat(workflows)**: Resolve component packages from bit.cloud registry

This commit introduces a step in the workflows to resolve component packages from the bit.cloud registry. The step configures npm to use the bit.cloud registry and sets the authentication token. This change is necessary for component installation using package managers other than Bit.

💄 **:lipstick: style(workflows)**: Remove unnecessary conditional check in CLA Assistant

This commit removes an unnecessary conditional check in the CLA workflow. The conditional check on the `cla_enabled` environment variable is removed, as it is redundant and does not affect the workflow's functionality.

♻️ **:recycle: refactor(workflows)**: Cleanup and standardize workflow files

This commit standardizes and cleans up various workflow files. The changes ensure consistency and improve readability across the workflow files. No functional changes are included in this commit.

🔨 **:hammer: fix(workflows)**: Update file modification date in GitHub Management workflow

This commit updates the last modified date in the header comment of the GitHub Management workflow file. The modification date is corrected to reflect the actual date of the file modification.

🐛 **:bug: fix(workflows)**: Use correct workflow telemetry action in GitGuardian IAC scan

This commit fixes a mistake in the GitGuardian IAC scan workflow. The workflow telemetry action was using an incorrect version. The action is now correctly set to `catchpoint/workflow-telemetry-action@v1.8.7` for better monitoring and insights.

🎨 **:art: style(workflows)**: Update header comment in Lane Cleanup workflow

This commit updates the header comment in the Lane Cleanup workflow file to include relevant information. The header comment now includes the project, file path, author information, and license details. The modification date is also updated to reflect the latest changes.

Signed-off-by: Jonathan Stevens <jonathan.stevens@eventiva.co.uk>

restyled-commits committed:
Restyled by prettier-markdown

restyled-commits committed:
Restyled by prettier-yaml

EventivaLtd commented:
## Workflow Telemetry - Github Management / Release Mastermind
Workflow telemetry for commit [e5534b25b93ef4b65ebadfc76bd91f6ed3c0b349](https://github.com/Eventiva/Eventiva/commit/e5534b25b93ef4b65ebadfc76bd91f6ed3c0b349)
You can access workflow job details [here](https://github.com/Eventiva/Eventiva/runs/19574969692?check_suite_focus=true)

### Step Trace

```mermaid
gantt
	title Release Mastermind
	dateFormat x
	axisFormat %H:%M:%S
	Set up job : milestone, 1702416127000, 1702416129000
	Workflow Telemetry : 1702416129000, 1702416129000
	Release Mastermind : 1702416129000, 1702416133000

```
### CPU Metrics
![chart_stacked_area_time_c425ddf3-1e66-4205-a066-eabc21b4a76a](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_c425ddf3-1e66-4205-a066-eabc21b4a76a)

### Memory Metrics
![chart_stacked_area_time_3d288255-6cbd-4c70-8353-ab8abdaef5de](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_3d288255-6cbd-4c70-8353-ab8abdaef5de)

### IO Metrics
|               | Read      | Write     |
|---            |---        |---        |
| Network I/O   | ![chart_line_time_eeb0b507-f79c-4937-bfa4-8d4c6e7bc085](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_eeb0b507-f79c-4937-bfa4-8d4c6e7bc085)        | ![chart_line_time_1a8e5127-39e2-4beb-a363-4f6f85495df9](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_1a8e5127-39e2-4beb-a363-4f6f85495df9)        |
| Disk I/O      | ![chart_line_time_8467a2e7-33c4-471b-b5e8-0aa94e7c856d](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_8467a2e7-33c4-471b-b5e8-0aa94e7c856d)              | ![chart_line_time_30946c7b-610d-41b4-90f1-c5ed51a1dd31](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_30946c7b-610d-41b4-90f1-c5ed51a1dd31)              |

### Process Trace

#### Top 100 processes with highest duration

```mermaid
gantt
	title Release Mastermind
	dateFormat x
	axisFormat %H:%M:%S
	node (smartcloud) : 1702416129633, 1702416133021
	sh : 1702416129661, 1702416129663
	ip : 1702416129663, 1702416129664
	grep : 1702416129663, 1702416129664
	sh : 1702416129668, 1702416129672
	lsblk : 1702416129669, 1702416129672
	grep : 1702416129669, 1702416129672
	sh : 1702416129674, 1702416129683
	cat : 1702416129676, 1702416129676
	cat : 1702416129677, 1702416129677
	cat : 1702416129678, 1702416129678
	cat : 1702416129678, 1702416129678
	cat : 1702416129679, 1702416129679
	cat : 1702416129680, 1702416129680
	cat : 1702416129681, 1702416129683
	sh : 1702416129681, 1702416129684
	cat : 1702416129682, 1702416129683
	egrep : 1702416129683, 1702416129684
	grep : 1702416129683, 1702416129684
	sh : 1702416133561, 1702416133563
	ip : 1702416133562, 1702416133563
	grep : 1702416133563, 1702416133564
	sh : 1702416133565, 1702416133569
	lsblk : 1702416133567, 1702416133569
	grep : 1702416133567, 1702416133569
	sh : 1702416133569, 1702416133579
	cat : 1702416133570, 1702416133570
	cat : 1702416133571, 1702416133571
	cat : 1702416133572, 1702416133572
	cat : 1702416133573, 1702416133573
	cat : 1702416133575, 1702416133575
	cat : 1702416133576, 1702416133576
	sh : 1702416133576, 1702416133580
	cat : 1702416133577, 1702416133578
	cat : 1702416133578, 1702416133579
	egrep : 1702416133578, 1702416133579
	grep : 1702416133578, 1702416133579

```

EventivaLtd commented:
## Workflow Telemetry - GitGuardian scan / GitGuardian scan
Workflow telemetry for commit [e5534b25b93ef4b65ebadfc76bd91f6ed3c0b349](https://github.com/Eventiva/Eventiva/commit/e5534b25b93ef4b65ebadfc76bd91f6ed3c0b349)
You can access workflow job details [here](https://github.com/Eventiva/Eventiva/runs/19574969791?check_suite_focus=true)

### Step Trace

```mermaid
gantt
	title GitGuardian scan
	dateFormat x
	axisFormat %H:%M:%S
	Set up job : milestone, 1702416127000, 1702416129000
	Pull gitguardian/ggshield-v1.22.0 : 1702416129000, 1702416135000
	Workflow Telemetry : 1702416135000, 1702416135000
	Checkout : 1702416135000, 1702416136000
	GitGuardian scan : 1702416137000, 1702416139000
	Post Checkout : 1702416139000, 1702416139000

```
### CPU Metrics
![chart_stacked_area_time_ea4f4bbe-6723-40c9-9f9a-a88f08a2aedf](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_ea4f4bbe-6723-40c9-9f9a-a88f08a2aedf)

### Memory Metrics
![chart_stacked_area_time_63550422-2121-460a-bc9b-f4c65dfaa5b6](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_63550422-2121-460a-bc9b-f4c65dfaa5b6)

### IO Metrics
|               | Read      | Write     |
|---            |---        |---        |
| Network I/O   | ![chart_line_time_b77fbb9b-a09c-4d6c-b7ad-87ef7b6a355a](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_b77fbb9b-a09c-4d6c-b7ad-87ef7b6a355a)        | ![chart_line_time_0070a406-89d5-4ac6-8251-bf7e1e6a4e11](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_0070a406-89d5-4ac6-8251-bf7e1e6a4e11)        |
| Disk I/O      | ![chart_line_time_cc799bc4-9dda-46fc-9d00-c9026e0f9351](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_cc799bc4-9dda-46fc-9d00-c9026e0f9351)              | ![chart_line_time_a7181b24-8de4-4c25-85fa-646d8537ef4f](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_a7181b24-8de4-4c25-85fa-646d8537ef4f)              |

### Process Trace

#### Top 100 processes with highest duration

```mermaid
gantt
	title GitGuardian scan
	dateFormat x
	axisFormat %H:%M:%S
	node (checkout) : 1702416135549, 1702416136980
	sh : 1702416135571, 1702416135573
	ip : 1702416135572, 1702416135573
	grep : 1702416135572, 1702416135573
	sh : 1702416135577, 1702416135581
	lsblk : 1702416135579, 1702416135582
	grep : 1702416135579, 1702416135582
	sh : 1702416135584, 1702416135592
	sh : 1702416135590, 1702416135594
	cat : 1702416135591, 1702416135593
	cat : 1702416135592, 1702416135593
	egrep : 1702416135592, 1702416135593
	grep : 1702416135592, 1702416135593
	git : 1702416135644, 1702416135648
	git : 1702416135655, 1702416135656
	git : 1702416135661, 1702416135667
	git : 1702416135669, 1702416135670
	git : 1702416135674, 1702416135675
	git : crit, 1702416135678, 1702416135679
	git : 1702416135682, 1702416135709
	git-submodule : 1702416135684, 1702416135709
	sed : 1702416135685, 1702416135686
	git : 1702416135687, 1702416135691
	git : 1702416135699, 1702416135700
	git : 1702416135701, 1702416135702
	git : 1702416135703, 1702416135704
	git : 1702416135704, 1702416135705
	git : 1702416135705, 1702416135706
	git : 1702416135709, 1702416135710
	git : crit, 1702416135712, 1702416135713
	git : 1702416135716, 1702416135738
	git-submodule : 1702416135718, 1702416135738
	git : 1702416135728, 1702416135729
	git : 1702416135730, 1702416135731
	git : 1702416135731, 1702416135732
	git : 1702416135732, 1702416135733
	git : 1702416135734, 1702416135735
	git : 1702416135737, 1702416135738
	git : 1702416135741, 1702416135742
	git : 1702416135746, 1702416136677
	git : 1702416135749, 1702416136281
	git-remote-http : 1702416135750, 1702416136280
	git : 1702416136042, 1702416136661
	git : 1702416136662, 1702416136669
	git : 1702416136676, 1702416136677
	git : 1702416136681, 1702416136770
	git : 1702416136774, 1702416136775
	git : crit, 1702416136779, 1702416136780
	git : 1702416136783, 1702416136784
	git : 1702416136787, 1702416136788
	git : 1702416136791, 1702416136813
	git-submodule : 1702416136793, 1702416136813
	sed : 1702416136794, 1702416136795
	git : 1702416136803, 1702416136804
	git : 1702416136804, 1702416136805
	git : 1702416136806, 1702416136807
	git : 1702416136807, 1702416136808
	git : 1702416136809, 1702416136810
	git : 1702416136812, 1702416136813
	git : 1702416136816, 1702416136838
	git-submodule : 1702416136818, 1702416136838
	git : 1702416136820, 1702416136821
	git : 1702416136827, 1702416136828
	git : 1702416136829, 1702416136830
	git : 1702416136831, 1702416136832
	git : 1702416136832, 1702416136833
	git : 1702416136834, 1702416136835
	sed : 1702416136835, 1702416136836
	git : 1702416136837, 1702416136838
	git : 1702416136841, 1702416136863
	git-submodule : 1702416136843, 1702416136863
	sed : 1702416136844, 1702416136845
	git : 1702416136853, 1702416136854
	git : 1702416136855, 1702416136856
	git : 1702416136856, 1702416136857
	git : 1702416136858, 1702416136859
	git : 1702416136859, 1702416136860
	git : 1702416136862, 1702416136863
	git : 1702416136866, 1702416136888
	git-submodule : 1702416136868, 1702416136888
	git : 1702416136878, 1702416136879
	git : 1702416136880, 1702416136881
	git : 1702416136891, 1702416136913
	git-submodule : 1702416136893, 1702416136913
	git : 1702416136916, 1702416136937
	git-submodule : 1702416136918, 1702416136938
	git : 1702416136941, 1702416136962
	git-submodule : 1702416136943, 1702416136963
	docker : 1702416137012, 1702416139262
	node (checkout) : 1702416139275, 1702416139438
	git : 1702416139374, 1702416139396
	git-submodule : 1702416139376, 1702416139396
	git : 1702416139408, 1702416139430
	git-submodule : 1702416139410, 1702416139431
	sh : 1702416139831, 1702416139833
	sh : 1702416139835, 1702416139838
	lsblk : 1702416139836, 1702416139838
	grep : 1702416139837, 1702416139839
	sh : 1702416139838, 1702416139845
	sh : 1702416139846, 1702416139850

```

EventivaLtd commented:
## Workflow Telemetry - Github Management / Release Mastermind
Workflow telemetry for commit [09bc25565e26fe70c8d2324a7939ef4741e6da84](https://github.com/Eventiva/Eventiva/commit/09bc25565e26fe70c8d2324a7939ef4741e6da84)
You can access workflow job details [here](https://github.com/Eventiva/Eventiva/runs/19575066193?check_suite_focus=true)

### Step Trace

```mermaid
gantt
	title Release Mastermind
	dateFormat x
	axisFormat %H:%M:%S
	Set up job : milestone, 1702416347000, 1702416349000
	Workflow Telemetry : 1702416349000, 1702416349000

```
### CPU Metrics
![chart_stacked_area_time_ebb5c5e9-4dd1-4daf-b66f-73ad93417fad](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_ebb5c5e9-4dd1-4daf-b66f-73ad93417fad)

### Memory Metrics
![chart_stacked_area_time_012e0908-6481-47d8-9893-39062bd289fe](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_012e0908-6481-47d8-9893-39062bd289fe)

### IO Metrics
|               | Read      | Write     |
|---            |---        |---        |
| Network I/O   | ![chart_line_time_804a8a1e-8ded-47db-9f76-e7c9442d1128](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_804a8a1e-8ded-47db-9f76-e7c9442d1128)        | ![chart_line_time_d1906f83-644c-4b52-a153-061ba6e7335e](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_d1906f83-644c-4b52-a153-061ba6e7335e)        |
| Disk I/O      | ![chart_line_time_fb00be2a-cf76-441c-91d1-80dd53e51457](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_fb00be2a-cf76-441c-91d1-80dd53e51457)              | ![chart_line_time_67210a37-d6bb-436c-82c6-bac189aeb9b7](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_67210a37-d6bb-436c-82c6-bac189aeb9b7)              |

### Process Trace

#### Top 100 processes with highest duration

```mermaid
gantt
	title Release Mastermind
	dateFormat x
	axisFormat %H:%M:%S
	node (smartcloud) : 1702416349536, 1702416351330
	sh : 1702416349552, 1702416349554
	ip : 1702416349553, 1702416349554
	grep : 1702416349553, 1702416349554
	sh : 1702416349557, 1702416349562
	lsblk : 1702416349559, 1702416349563
	grep : 1702416349559, 1702416349563
	sh : 1702416349563, 1702416349572
	cat : 1702416349565, 1702416349566
	cat : 1702416349566, 1702416349567
	cat : 1702416349567, 1702416349568
	cat : 1702416349569, 1702416349569
	sh : 1702416349569, 1702416349573
	cat : 1702416349570, 1702416349571
	egrep : 1702416349570, 1702416349571
	cat : 1702416349570, 1702416349571
	grep : 1702416349570, 1702416349571
	cat : 1702416349571, 1702416349571
	cat : 1702416349572, 1702416349572
	sh : 1702416351694, 1702416351696
	ip : 1702416351695, 1702416351696
	grep : 1702416351695, 1702416351696
	sh : 1702416351699, 1702416351703
	lsblk : 1702416351700, 1702416351703
	grep : 1702416351700, 1702416351703
	sh : 1702416351702, 1702416351709
	cat : 1702416351703, 1702416351703
	cat : 1702416351704, 1702416351704
	cat : 1702416351705, 1702416351705
	cat : 1702416351706, 1702416351706
	cat : 1702416351707, 1702416351707
	cat : 1702416351708, 1702416351708
	cat : 1702416351709, 1702416351709
	sh : 1702416351711, 1702416351714
	cat : 1702416351712, 1702416351712
	egrep : 1702416351713, 1702416351714
	grep : 1702416351713, 1702416351714

```

EventivaLtd commented:
## Workflow Telemetry - GitGuardian scan / GitGuardian scan
Workflow telemetry for commit [09bc25565e26fe70c8d2324a7939ef4741e6da84](https://github.com/Eventiva/Eventiva/commit/09bc25565e26fe70c8d2324a7939ef4741e6da84)
You can access workflow job details [here](https://github.com/Eventiva/Eventiva/runs/19575066282?check_suite_focus=true)

### Step Trace

```mermaid
gantt
	title GitGuardian scan
	dateFormat x
	axisFormat %H:%M:%S
	Set up job : milestone, 1702416347000, 1702416349000
	Pull gitguardian/ggshield-v1.22.0 : 1702416349000, 1702416355000
	Workflow Telemetry : 1702416355000, 1702416355000
	Checkout : 1702416356000, 1702416357000
	GitGuardian scan : 1702416357000, 1702416359000
	Post Checkout : 1702416360000, 1702416360000

```
### CPU Metrics
![chart_stacked_area_time_51fe05a5-5612-47e1-9ad5-fea33f80233e](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_51fe05a5-5612-47e1-9ad5-fea33f80233e)

### Memory Metrics
![chart_stacked_area_time_2aae3000-62da-4a69-8bbe-2589727e691a](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_2aae3000-62da-4a69-8bbe-2589727e691a)

### IO Metrics
|               | Read      | Write     |
|---            |---        |---        |
| Network I/O   | ![chart_line_time_465e88fa-f72e-46dc-a9ae-4df6953a6964](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_465e88fa-f72e-46dc-a9ae-4df6953a6964)        | ![chart_line_time_45dad639-8463-4a28-b3ed-b8627b7c8ba6](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_45dad639-8463-4a28-b3ed-b8627b7c8ba6)        |
| Disk I/O      | ![chart_line_time_a6b056f7-fb33-4cbd-9fc5-5d9e1d7a0fb8](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_a6b056f7-fb33-4cbd-9fc5-5d9e1d7a0fb8)              | ![chart_line_time_5f0bec0f-0ad3-4bbc-b854-7c9d86a2e174](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_5f0bec0f-0ad3-4bbc-b854-7c9d86a2e174)              |

### Process Trace

#### Top 100 processes with highest duration

```mermaid
gantt
	title GitGuardian scan
	dateFormat x
	axisFormat %H:%M:%S
	node (checkout) : 1702416355941, 1702416357389
	sh : 1702416355961, 1702416355963
	ip : 1702416355962, 1702416355963
	grep : 1702416355962, 1702416355963
	sh : 1702416355966, 1702416355971
	lsblk : 1702416355968, 1702416355972
	grep : 1702416355968, 1702416355972
	sh : 1702416355972, 1702416355981
	cat : 1702416355978, 1702416355979
	sh : 1702416355978, 1702416355983
	cat : 1702416355979, 1702416355980
	cat : 1702416355979, 1702416355980
	egrep : 1702416355979, 1702416355980
	grep : 1702416355979, 1702416355980
	git : 1702416356040, 1702416356045
	git : 1702416356052, 1702416356053
	git : 1702416356058, 1702416356064
	git : 1702416356068, 1702416356069
	git : 1702416356073, 1702416356074
	git : crit, 1702416356077, 1702416356078
	git : 1702416356082, 1702416356111
	git-submodule : 1702416356084, 1702416356111
	sed : 1702416356085, 1702416356086
	git : 1702416356087, 1702416356092
	git : 1702416356100, 1702416356101
	git : 1702416356102, 1702416356103
	git : 1702416356104, 1702416356105
	git : 1702416356105, 1702416356106
	git : 1702416356107, 1702416356108
	git : 1702416356110, 1702416356111
	git : crit, 1702416356114, 1702416356115
	git : 1702416356118, 1702416356140
	git-submodule : 1702416356120, 1702416356140
	git : 1702416356130, 1702416356131
	git : 1702416356132, 1702416356133
	git : 1702416356133, 1702416356134
	git : 1702416356135, 1702416356136
	git : 1702416356136, 1702416356137
	git : 1702416356140, 1702416356141
	git : 1702416356144, 1702416356145
	git : 1702416356149, 1702416357072
	git : 1702416356152, 1702416356722
	git-remote-http : 1702416356153, 1702416356721
	git : 1702416356427, 1702416357055
	git : 1702416357056, 1702416357063
	git : 1702416357072, 1702416357073
	git : 1702416357077, 1702416357167
	git : 1702416357171, 1702416357172
	git : crit, 1702416357176, 1702416357177
	git : 1702416357180, 1702416357181
	git : 1702416357184, 1702416357185
	git : 1702416357188, 1702416357211
	git-submodule : 1702416357190, 1702416357211
	sed : 1702416357191, 1702416357192
	sed : 1702416357195, 1702416357196
	git : 1702416357200, 1702416357201
	git : 1702416357202, 1702416357203
	git : 1702416357203, 1702416357204
	git : 1702416357205, 1702416357206
	git : 1702416357206, 1702416357207
	sed : 1702416357208, 1702416357209
	git : 1702416357210, 1702416357211
	git : 1702416357214, 1702416357237
	git-submodule : 1702416357217, 1702416357238
	git : 1702416357227, 1702416357228
	git : 1702416357229, 1702416357230
	git : 1702416357230, 1702416357231
	git : 1702416357232, 1702416357233
	git : 1702416357233, 1702416357234
	git : 1702416357237, 1702416357238
	git : 1702416357241, 1702416357267
	git-submodule : 1702416357243, 1702416357267
	git : 1702416357256, 1702416357257
	git : 1702416357258, 1702416357259
	git : 1702416357259, 1702416357260
	git : 1702416357261, 1702416357262
	git : 1702416357263, 1702416357264
	sed : 1702416357265, 1702416357266
	git : 1702416357266, 1702416357267
	git : 1702416357270, 1702416357292
	git-submodule : 1702416357272, 1702416357293
	sed : 1702416357273, 1702416357274
	git : 1702416357296, 1702416357318
	git-submodule : 1702416357299, 1702416357319
	git : 1702416357322, 1702416357344
	git-submodule : 1702416357324, 1702416357345
	git : 1702416357348, 1702416357371
	git-submodule : 1702416357350, 1702416357371
	docker : 1702416357422, 1702416359811
	node (checkout) : 1702416359823, 1702416359997
	git : 1702416359930, 1702416359953
	git-submodule : 1702416359932, 1702416359954
	git : 1702416359966, 1702416359990
	git-submodule : 1702416359968, 1702416359990
	sh : 1702416360362, 1702416360364
	sh : 1702416360366, 1702416360369
	lsblk : 1702416360368, 1702416360370
	grep : 1702416360368, 1702416360370
	sh : 1702416360370, 1702416360377
	sh : 1702416360379, 1702416360383

```

EventivaLtd commented:
## Workflow Telemetry - Github Management / Release Mastermind
Workflow telemetry for commit [09bc25565e26fe70c8d2324a7939ef4741e6da84](https://github.com/Eventiva/Eventiva/commit/09bc25565e26fe70c8d2324a7939ef4741e6da84)
You can access workflow job details [here](https://github.com/Eventiva/Eventiva/runs/19575127435?check_suite_focus=true)

### Step Trace

```mermaid
gantt
	title Release Mastermind
	dateFormat x
	axisFormat %H:%M:%S
	Set up job : milestone, 1702416464000, 1702416466000
	Workflow Telemetry : 1702416466000, 1702416466000
	Release Mastermind : 1702416466000, 1702416468000

```
### CPU Metrics
![chart_stacked_area_time_c0f94172-eda3-43b8-8374-d60909556146](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_c0f94172-eda3-43b8-8374-d60909556146)

### Memory Metrics
![chart_stacked_area_time_6986bc52-c4d6-462e-8fd5-49b31f30398e](https://api.globadge.com/v1/chartgen/stacked-area/time/chart_stacked_area_time_6986bc52-c4d6-462e-8fd5-49b31f30398e)

### IO Metrics
|               | Read      | Write     |
|---            |---        |---        |
| Network I/O   | ![chart_line_time_9fa59630-5047-4144-9095-1b18f1b29f83](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_9fa59630-5047-4144-9095-1b18f1b29f83)        | ![chart_line_time_95f9f102-8392-460b-b973-b2da2b2b32ac](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_95f9f102-8392-460b-b973-b2da2b2b32ac)        |
| Disk I/O      | ![chart_line_time_619880c3-84d5-4a7c-8a4c-7eae8c5ca1dd](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_619880c3-84d5-4a7c-8a4c-7eae8c5ca1dd)              | ![chart_line_time_bc914d2c-fa80-4622-8fff-a247a73a42d4](https://api.globadge.com/v1/chartgen/line/time/chart_line_time_bc914d2c-fa80-4622-8fff-a247a73a42d4)              |

### Process Trace

#### Top 100 processes with highest duration

```mermaid
gantt
	title Release Mastermind
	dateFormat x
	axisFormat %H:%M:%S
	node (smartcloud) : 1702416466356, 1702416468199
	sh : 1702416466378, 1702416466380
	ip : 1702416466379, 1702416466380
	grep : 1702416466379, 1702416466380
	sh : 1702416466383, 1702416466388
	lsblk : 1702416466385, 1702416466388
	grep : 1702416466385, 1702416466389
	sh : 1702416466389, 1702416466398
	cat : 1702416466391, 1702416466392
	cat : 1702416466392, 1702416466392
	cat : 1702416466393, 1702416466393
	cat : 1702416466395, 1702416466396
	sh : 1702416466395, 1702416466399
	cat : 1702416466396, 1702416466397
	egrep : 1702416466396, 1702416466397
	cat : 1702416466396, 1702416466397
	grep : 1702416466396, 1702416466397
	cat : 1702416466398, 1702416466399
	cat : 1702416466399, 1702416466399
	sh : 1702416468550, 1702416468552
	ip : 1702416468552, 1702416468553
	grep : 1702416468552, 1702416468553
	sh : 1702416468555, 1702416468558
	lsblk : 1702416468556, 1702416468558
	grep : 1702416468556, 1702416468558
	sh : 1702416468558, 1702416468566
	cat : 1702416468559, 1702416468559
	cat : 1702416468560, 1702416468560
	cat : 1702416468561, 1702416468561
	cat : 1702416468562, 1702416468562
	cat : 1702416468563, 1702416468563
	cat : 1702416468564, 1702416468564
	cat : 1702416468565, 1702416468567
	sh : 1702416468565, 1702416468568
	cat : 1702416468566, 1702416468567
	egrep : 1702416468566, 1702416468567
	grep : 1702416468566, 1702416468567

```

TGTGamer committed:
🔧 chore(workflows): disable PR comments for telemetry actions in GitHub workflows

The PR comments for telemetry actions in the GitHub workflows have been disabled by setting the `comment_on_pr` flag to `false`. This change was made to reduce noise in the PR discussions and improve the overall workflow experience.
